test/storage/garbage_collector_test.cpp

```diff
@@ -1269,16 +1283,16 @@ TEST_F(GarbageCollectorTests, DeleteTest3) {
   }
 }
 
+/*
+ * Consider performing GC on the version chain sorted from newest to oldest:
+ * DELETE
+ * U2
+ * U1
+ * INSERT
+ * txn1 <- active txn
```

Comment:
Consistency with the previous notation
```suggestion
 T1 <- active txn
```